ui/ops/src/traits/health/HealthCheckCard.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-card elevation="0">
+    <v-card-text class="text-body-1 py-0 my-2">{{ name }}</v-card-text>
+    <v-card-text class="text-body-2 py-0 my-2">{{ description }}</v-card-text>
+    <v-card-text class="py-0">
+      <normality-time-text :model-value="props.modelValue"/>
+    </v-card-text>
+    <v-card-text>
+      <reliability-time-text :model-value="props.modelValue"/>
+    </v-card-text>
+    <v-card-text v-if="hasBounds">
+      <h4 class="text-caption">Measured value:</h4>
+      <bounds-text :model-value="props.modelValue" class="ml-1"/>
+    </v-card-text>
+    <v-card-text>
+      <h4 class="text-caption">Impact:</h4>
```

Comment:
I worry the presentation of this info is misleading. In the attached image, it looks like we have a fail on BS 5266. It's not obvious that the impact is conditional on the check being not 'Normal' above. The fact that 'fail' is highlighted in red makes this problem worse.

Ideas:
- Grey out the impact when the check is Normal
- Rename the section to 'Potential Impact' vs. 'Current Impact' based on the check status
- Rearrange the widget to clearly link the impacts to the normality
<img width="301" height="299" alt="2025-11-28 Health misleading impact" src="https://github.com/user-attachments/assets/5c3e564c-6e23-4b4a-b294-6725f2688af5" />